assets/js/utils/rtcUtils.ts

```diff
@@ -140,11 +190,18 @@ export const connectWebrtc = async (
               },
             };
           });
+
+          if ("enabled" in metadata) track.enabled = metadata.enabled;
+          updateMicAndCamStatusCallback();
         }
       },
-      onTrackAdded: () => {
-        // do nothing
+      onTrackUpdated({ track, metadata }: TrackContext) {
+        if (track != null) {
+          if ("enabled" in metadata) track.enabled = metadata.enabled;
+          updateMicAndCamStatusCallback();
```

Comment:
What is the sense of updating the mic and cam status if we didn't change `track.enabled`?

Reply:
If nothing changed, his function has no use. I thought there could possibly be a change of `track.enabled` property that is not from webrtc, but there is no such scenario.